.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,36 @@
+name: pipeline
+on: pull_request
+
+jobs:
+  pipeline:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
+
+      - name: Cache Vendors
```

Comment:
Instead of using this, just use Ramsey/composer-install action

Reply:
thanks for that pointer :) see #4